empty/README.md

```diff
@@ -15,7 +16,7 @@ Run `temporal server start-dev` to start [Temporal Server](https://github.com/te
 
 The `package.json` file contains scripts for running the client, the Worker, and tests.
 
-1. In a shell, run `npm run start.watch` to start the Worker and reload it when code changes..
+1. In a shell, run `npm run start.watch` to start the Worker and reload it when code changes.
```

Comment:
Do we need to mention 'reload it when code changes' when we have `start.watch`? Because of the .watch, it should automatically detect code changes.

Reply:
The sentence may be poorly constructed. But it's saying that the command "starts the worker and reloads it".

Reply:
Oh gotcha, I thought that you meant to tell the user to reload that command every time there is a change